docs/customize/s3.md

```diff
@@ -2,7 +2,7 @@
 
 ## I chose S3 when initializing the repository!
 
-If you are reading this section, you are lucky! You will only have to perform a few configuration changes. If on the contrary you didn't choose S3 as storage when initializing the repository file system, [navigate below](#i-didnt-choose-s3-when-initializing) to follow more in depth instructions!
+If you are reading this section, you are lucky! You will only have to perform a few configuration changes.
```

Comment:
I think we should add a warning like "this section assumes you choose S3 when bootstrapping the instance". Because if not, the docs should mention things like "add container to docker-compose" etc.